program_intent_engagement/apps/api/v1/views.py

```diff
@@ -1 +1,61 @@
-# Create your views here.
+"""
+V1 API Views
+"""
+from rest_framework import status
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from edx_rest_framework_extensions.auth.jwt.authentication import JwtAuthentication
+from django.db.models import F, Window
+from django.db.models.functions import RowNumber
+
+from program_intent_engagement.apps.core.models import ProgramIntent
+from program_intent_engagement.apps.api.serializers import ProgramIntentSerializer
+
+
+
+
+# TODO Do I add an @schema thing here?
```

Comment:
Something like [this](https://github.com/edx/edx-exams/blob/f0fbecf17b76666029a502023e551e8f891cf1fe/edx_exams/apps/api/v1/views.py#L144) in the edx repo

Reply:
Ah, that's a decorator for adding documentation to the class. You can read about it [here](https://github.com/openedx/api-doc-tools). I don't think we have api doc tools set up for the repository yet, so this can be done in a later ticket or when we need it.

Reply:
@m-pitera Varsha created this ticket: https://2u-internal.atlassian.net/browse/MST-1558, you could add this endpoint to the ticket details

Reply:
got it, done 👍